EIPS/eip-1538.md

```diff
@@ -9,7 +9,7 @@ category: ERC
 created: 2018-10-31
 ---
 
-Replaced by [EIP-2535 Diamond Standard](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2535.md).
+Replaced by [EIP-2535 Diamond Standard](./eip-2535.md).
```

Comment:
Why is this in the body of the EIP rather than as the [preamble field](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md#eip-header-preamble) `superceded-by`?

Reply:
Because EIP-2535 isn't final yet.   But I would be happy to make this change if it is okay.

Reply:
Ah, I see. I suppose this is okay for now then. Looking over EIP-1 now and I don't believe it covers this sort of situation very well.

Reply:
Also, there is talk of removing the superceded by concept, since deciding what supercedes what is a governance nightmare.  Take for example the 15 different token standards that all claim to supersede ERC-20.

Reply:
I'm in favor of removing it.